packages/fairchem-demo-ocpapi/pyproject.toml

```diff
@@ -17,12 +17,12 @@ classifiers = [
     "Programming Language :: Python :: 3",
 ]
 dependencies = [
-    "dataclasses-json == 0.6.0",
-    "inquirer == 3.1.3",
-    "requests == 2.32.0",
-    "responses == 0.23.2",
-    "tenacity == 8.2.3",
-    "tqdm == 4.66.1",
+    "dataclasses-json >= 0.6.0",
```

Comment:
What do you think about putting an upper bound on these to prevent them from going to higher major versions? Just to try and avoid breaking changes for projects that use semantic versioning.